app/controllers/portal/client_logins_controller.rb

```diff
@@ -63,11 +68,18 @@ def check_verification_code
 
     def account_locked; end
 
+    def unlock_matching_records
+      @records.each(&:unlock_for_login!)
+    end
```

Comment:
this doesn't do anything -- b/c `lock_expired?` always returns false for a locked `Client` unless `unlock_strategy_enabled?(:time)` is true. Clients are manually unlocked by their org leads/site coordinators. Thinking about removing this & leaving a comment here instead

Reply:
removed & leaving a comment